packages/utils/src/types.ts

```diff
@@ -750,7 +750,7 @@ export interface WidgetProps<T = any, S extends StrictRJSFSchema = RJSFSchema, F
   /** The input blur event handler; call it with the widget id and value */
   onBlur: (id: string, value: any) => void;
   /** The value change event handler; call it with the new value every time it changes */
-  onChange: (value: any) => void;
+  onChange: (value: any, es?: ErrorSchema<T>, id?: string) => void;
```

Comment:
I don't see any code that grabs these errors from the widget and passes it up the chain. If you added tests for this you should be able to validate that a custom widget that return an error schema and id get those errors shown.